core-web/libs/edit-content/src/lib/fields/dot-edit-content-custom-field/dot-edit-content-custom-field.component.scss

```
@@ -0,0 +1,25 @@
+@use "variables" as *;
+
+iframe {
+    display: block;
+    width: 100%;
+}
+
+.legacy-custom-field {
+    &--fullscreen {
```

He means the class name.

```suggestion
.legacy-custom-field--fullscreen {
```